fikretellek/tech-products-demo
# Auto-publish suggestions if suggested by admin

This is a:






### Description



- **Purpose** -  allowing auto-publishing suggestions written by an admin 



- **How to check** - 
  - go to home page
  - sign in as an admin
  - go to suggest
    - enter title
    - enter url
    - click to suggest button
  - go to home page
  - your suggestion should be visible in the published resources list.
  - if you sign in as a user and do the steps, your suggestion should NOT be in the list.
 
  - 

### Links

[Automatic publication of admins' resource suggestions #11](https://github.com/CodeYourFuture/tech-products-demo/issues/11)


### Author checklist
